### PR TITLE
Adding features

### PR DESCRIPTION
**Pull request by Chankster**
Please review.  Especially the flair sections as I was unable to test the 'i-' flair crap in /r/hardwareswap.  I know the non 'i-' version works (as its currently running).

### CODE REVIEW

**Comment by nathaniel-ortiz** [comment]
I can easily write up some css to switch over from the i- flair format to just numbers, the issue is going back through and changing all the user flairs.

Is there any way to automate the process?

**Comment by Chankster** [comment]
Actually changing the flair is the easy part with the new scripts I added.  Just dump the current flair as json using modutils, find/replace the 'i-' in the output json, use the flair_sub_import.py script to reimport it back.

**Comment by nathaniel-ortiz** [comment]
nice.

you guys mind if i clone the flair?

won't do it without your guys's approval of course.

File: post_check.py [hunk 1]
```diff
@@ -97,6 +106,25 @@ def main():
...
+                            row = curs.fetchone()
+
+                            # ensure that time of last post is > 24hrs
+                            if row is not None:
+                                if not row['lastid']:
+                                    lastid = ""
+                                else:
+                                    lastid = row['lastid']
+                                if row['lastpost']:
+                                    if (((((datetime.utcnow() - row['lastpost']).total_seconds() / 3600) < 24) and (((datetime.utcnow() - row['lastpost']).total_seconds() / 60) > 10)) and (lastid != "") and (post.id != lastid) and not post.approved_by):
```
Comment (soilheart) [inline post_check.py]: Please make the time (24) a setting.

Comment (Chankster) [reply post_check.py]: Done.  Originally left it hard coded because I wasn't aware of any sub that let people post more frequently than 24 hour.

**Comment by Chankster** [comment]
@soilheart can you do one last final review before I merge?

**Comment by soilheart** [comment]
Oh, sorry, I seem to have missed your last message. Good call merge:ing it. I'll probably look at it in the coming weeks to see how we can use the new features, and maybe make a few minor changes here and there.

**Pull request merged**